apps/angular/57-content-projection-default/src/app/card.component.ts

```diff
@@ -1,22 +1,15 @@
-import { ChangeDetectionStrategy, Component, input } from '@angular/core';
+import { ChangeDetectionStrategy, Component } from '@angular/core';
 
 @Component({
   selector: 'app-card',
   imports: [],
   template: `
-    <div>{{ title() }}</div>
-    @if (message()) {
-      <div>{{ message() }}</div>
-    } @else {
-      <div>Aucun message</div>
-    }
+    <ng-content select=".title"></ng-content>
+    <ng-content select=".message">Aucun message</ng-content>
```

Comment:
better but I advice you not to rely on css class for selector.
You should use specific selectors.

Reply:
I didn't wanna change the html but I have updated it a bit